tensorflow_addons/callbacks/tqdm_progress_bar.py

```diff
@@ -186,6 +193,37 @@ def on_batch_end(self, batch, logs={}):
 
                 # update timestamp for last update
                 self.last_update_time = now
+        else:
+            self.logs = logs
+
+    def on_train_begin(self, logs=None):
```

Comment:
Don't forget to change the internal method names here like `self._on_call_begin` to self._initialize_progbar` etc.